### PR TITLE
Cosmetic changes to navigator_params

### PR DESCRIPTION
**Pull request by LowOrbitIonCannon**
As per @hamishwillee  suggestion from this PR: #13159 .

Maybe we should add, that Unmanned collision avoidance is ONLY and ALWAYS triggered if a UTM_Globalposition message is received.

Eg. the Two Failure Cases:
-Manned aviation that sends UTM_Global_position is detected (as Unmanned)
   If it has both Mavlink and ADSB it will be received twice.
-Unmanned aviation that does not send UTM_Global_position is not.
(UTM_Global_Position also needs to be received by mavlink on the Drone)

#13159

### CODE REVIEW

File: src/modules/navigator/navigator_params.c [hunk 1]
```diff
@@ -128,28 +128,24 @@ PARAM_DEFINE_FLOAT(NAV_MC_ALT_RAD, 0.8f);
...
  * @unit m
  * @min 500
- *
  * @group Mission
  */
 PARAM_DEFINE_FLOAT(NAV_TRAFF_A_RADM, 500);
 
 /**
- * Set NAV TRAFFIC AVOID RADIUS
+ * Set avoidance radius/distance for umanned vehicles (UTM_GLOBAL_POSITION).
```
Comment (mhkabir) [inline src/modules/navigator/navigator_params.c]:
```suggestion
 * Set avoidance radius/distance for unmanned vehicles (UTM_GLOBAL_POSITION).
```

File: src/modules/navigator/navigator_params.c [hunk 2]
```diff
@@ -128,28 +128,24 @@ PARAM_DEFINE_FLOAT(NAV_MC_ALT_RAD, 0.8f);
...
  *
- * Defines the Radius where NAV TRAFFIC AVOID is Called
- * For Unmanned Aviation
+ * Defines the avoidance radius/distance at which the avoidance action is triggered for umanned vehicles (UTM_GLOBAL_POSITION).
```
Comment (mhkabir) [inline src/modules/navigator/navigator_params.c]:
```suggestion
 * Defines the avoidance radius/distance at which the avoidance action is triggered for unmanned vehicles (UTM_GLOBAL_POSITION).
```

File: src/modules/navigator/navigator_params.c [hunk 3]
```diff
@@ -128,28 +128,24 @@ PARAM_DEFINE_FLOAT(NAV_MC_ALT_RAD, 0.8f);
 PARAM_DEFINE_INT32(NAV_TRAFF_AVOID, 1);
 
 /**
- * Set NAV TRAFFIC AVOID RADIUS MANNED
+ * Set avoidance radius/distance for manned vehicles (ADS-B/FLARM).
```
Comment (dagar) [inline src/modules/navigator/navigator_params.c]: I would keep this relatively short for QGC.

Screenshot from 2020-02-02 12-52-07

Comment (hamishwillee) [reply src/modules/navigator/navigator_params.c]: Could perhaps change these to

> Set ADS-B/FLARM avoidance radius.

 and 
> 
> Set UTM avoidance radius.

The extended description covers the detail.

Comment (LowOrbitIonCannon) [reply src/modules/navigator/navigator_params.c]: I have no opinion on the optimal solution

**Comment by dagar** [comment]
Closing because the source branch disappeared (which chokes Jenkins).

**Pull request closed without merging**